modules/infrastructure-moving-router.adoc

```diff
@@ -81,9 +75,9 @@ Update the `nodeSelector` stanza to reference the `infra` label as shown:
 ----
 $ oc get pod -n openshift-ingress
 
-NAME                              READY     STATUS        RESTARTS   AGE       IP           NODE                           NOMINATED NODE
-router-default-86798b4b5d-bdlvd   1/1       Running       0          28s       10.130.2.4   ip-10-0-217-226.ec2.internal   <none>
-router-default-955d875f4-255g8    0/1       Terminating   0          19h       10.129.2.4   ip-10-0-148-172.ec2.internal   <none>
+NAME                              READY     STATUS        RESTARTS   AGE
+router-default-86798b4b5d-bdlvd   1/1       Running       0          28s
+router-default-955d875f4-255g8    0/1       Terminating   0          19h
```

Comment:
Above three lines can be reverted if using command `oc get pod -n openshift-ingress -o wide`.
And in latest 4.1 build the output like:
```
$ oc -n openshift-ingress get pod -o wide
NAME                              READY   STATUS    RESTARTS   AGE     IP            NODE                                               NOMINATED NODE   READINESS GATES
router-default-66c797c795-7ls7x   1/1     Running   0          4h27m   10.128.2.12   ip-172-31-159-23.ap-northeast-2.compute.internal   <none>           <none>
router-default-66c797c795-jj9k6   1/1     Running   0          4h27m   10.128.2.10   ip-172-31-159-23.ap-northeast-2.compute.internal   <none>           <none>
```
Please note one extra column named `READINESS GATES` is added.

Reply:
Thank you @lihongan! I've updated the command. Will you please check?

Reply:
LGTM.  Thank you @kalexand-rh